nova_rover_demos/pointCloudProcessing/ptcloudgen.py

```diff
@@ -0,0 +1,131 @@
+'''
+ptcloudgen.py 
+
+A class that generates dummy point clouds in accordance with user configurations. 
+The point clouds are designed to be mappable to a surface in order to be coherently 
+processable by processes.
+
+Author: Jack McRobbie
+'''
+import math
+import numpy as np
+import random as rn
+from matplotlib import pyplot
+from mpl_toolkits.mplot3d import Axes3D
+
+class point_cloud_gen:
+
+	def __init__ (self,size, features,noise = 1,N = 10000):
+		'''
+		size: the bounds on the range of the point cloud values. Is a list of n tuples where n is the number of dimensions
```

Comment:
Also probably best to indicate the scale (m/cm) of the upper and lower bound in size

Reply:
good point, ill make a change.